addons/l10n_se/data/account.account.template.csv

```diff
@@ -52,21 +52,17 @@
 "a1810","1810","Andel i börsnoterade företag","account.data_account_type_current_assets","l10nse_chart_template","False"
 "a1880","1880","Andra kortfristiga placeringar","account.data_account_type_current_assets","l10nse_chart_template","False"
 "a1890","1890","Nedskrivning av kortfristiga placceringar","account.data_account_type_current_assets","l10nse_chart_template","False"
-"a1910","1910","Kassa","account.data_account_type_liquidity","l10nse_chart_template","False"
-"a1920","1920","PlusGiro","account.data_account_type_liquidity","l10nse_chart_template","False"
-"a1930","1930","Företagskonto/checkkonto/affärskonto","account.data_account_type_liquidity","l10nse_chart_template","True"
-"a1940","1940","Övriga bankkonton","account.data_account_type_liquidity","l10nse_chart_template","False"
-"a2070","2070","Ändamålsbestämda medel","account.data_account_type_equity","l10nse_chart_template","False"
```

Comment:
1910, 1920, 1930, 1940 and 2070 is included as recommended minimum for BAS 2020.

Reply:
2070 is for "Eget kapital i ideella föreningar och stiftelser"
Then 2060 should also be included ?

Any comment @daniel-stenlov

Reply:
Don't touch it, it will interfere with the roadmap.

Reply:
@daniel-stenlov 
The question is if 2070 is included (not used by ltd companies)
should we include 2060 as well.
Not sure what roadmap we are talking about ?